lesson_03/script.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+const ul = document.getElementById("js-target");
+for (let i = 1; i <= 2; i += 1) {
+  const li = document.createElement("li");
+  const a = document.createElement("a");
+  const img = document.createElement("img");
+  a.text = "a" + i;
+  a.href = "a" + i + ".html";
```

Comment:
```suggestion
  a.textContent = `a${i}`;
  a.href = `a${i}.html`;
```
IMO
You can write like this.
https://qiita.com/azukiazusa/items/b0d4a816142134ea7f34